documentation/modules/rn-2.5.adoc

```diff
@@ -0,0 +1,199 @@
+
+.Migration of virtual machines with encrypted partitions fails during conversion
+
+vSphere only: Migrations from {rhv-short} and OpenStack don't fail, but the encryption key may be missing on the target OCP cluster.
+
+//inclusive language - cannot use execute
+.Migration fails during precopy/cutover while a snapshot operation is performed on the source VM
+
+////
+Some warm migrations from {rhv-short} might fail. When running a migration plan for warm migration of multiple VMs from {rhv-short}, the migrations of some VMs might fail during the cutover stage. In that case, restart the migration plan and set the cutover time for the VM migrations that failed in the first run.
+////
+Warm migration from {rhv-short} fails if a snapshot operation is performed on the source VM. If the user performs a snapshot operation on the source VM at the time when a migration snapshot is scheduled, the migration fails instead of waiting for the user’s snapshot operation to finish. link:https://issues.redhat.com/browse/MTV-456[(MTV-456)]
+
+.Unable to schedule migrated VM with multiple disks to more than one storage classes of type hostPath
+
+When migrating a VM with multiple disks to more than one storage classes of type hostPath, it may result in a VM that cannot be scheduled. Workaround: It is recommended to use shared storage on the target OCP cluster.
+
+.Non-supported guest operating systems in warm migrations
+
+Warm migrations and migrations to remote OCP clusters from vSphere do not support all types of guest operating systems that are supported in cold migrations to the local OCP cluster. It is a consequence of using RHEL 8 in the former case and RHEL 9 in the latter case. +
+See link:https://access.redhat.com/articles/1351473[Converting virtual machines from other hypervisors to KVM with virt-v2v in RHEL 7, RHEL 8, and RHEL 9] for the list of supported guest operating systems.
+
+.VMs from vSphere with RHEL 9 guest operating system may start with network interfaces that are down
+
+When migrating VMs that are installed with RHEL 9 as guest operating system from vSphere, their network interfaces could be disabled when they start in {ocp-name} Virtualization. link:https://issues.redhat.com/browse/MTV-491[(MTV-491)]
+
+.Import OVA: ConnectionTestFailed message appears when adding OVA provider
+
+When adding an OVA provider, the error message `ConnectionTestFailed` appears, although the provider is created successfully. After a few seconds, this message disappears. link:https://issues.redhat.com/browse/MTV-671[MTV-671]
+
+.OVA migration with multiple disks is not functioning
+
+OVA migration with multiple disks is not functioning as expected. When trying to migrate an OVA VM, with more than one disk, the migration becomes stuck at the `allocate disks` phase. link:https://issues.redhat.com/browse/MTV-676[MTV-676]
+
```

Comment:
@ahadas && @bkhizgiy && @qiyuann - adding the final two Known Issues 
Please can you add your comments, so we can have this ready for Thursday

Reply:
Looking good overall, regarding the first issue I think we need rephrase it a bit, because the issue can also indicate on a real connection problem (not part of the current bug) and not disappear in that case, we should write something like `"When adding an OVA provider, the error message ConnectionTestFailed may instantly appear, although the provider was created successfully. If after a few minutes the message isn't disappearing and the provider status turns to ready condition, the ova server pod creation really failed."`

Reply:
Change made